### PR TITLE
Feat/service account

### PR DESCRIPTION
**Pull request by KnechtionsCoding**

### CODE REVIEW

**Comment by KnechtionsCoding** [comment]
Provides documentation for #48 and provides support for serviceAccounts for #22 Image Pull Secrets are better managed that way.

File: pkg/resources/jobs/runner.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ type Script struct {
 // NewRunnerJob creates a new k6 job from a CRD
 func NewRunnerJob(k6 *v1alpha1.K6, index int) (*batchv1.Job, error) {
 	name := fmt.Sprintf("%s-%d", k6.Name, index)
-	command := []string{"scuttle", "k6", "run", "--quiet"}
```
Comment (dgzlopes) [inline pkg/resources/jobs/runner.go:26]: Do we want to remove the `--quiet` flag?

File: README.md [hunk 2]
```diff
@@ -131,6 +174,119 @@ Note that we are replacing the test job image (`k6-prometheus:latest`), passing
 (`--out prometheus`), and also exposing the ports required for Prometheus to scrape the metrics
 (in this case, that's port `5656`)
 
+You'll also need to create a pod monitor:
```
Comment (dgzlopes) [inline README.md:177]: This only applies if the user is using the Prometheus Operator. Right?

I'm not very familiar with it...

**Review by dgzlopes — COMMENTED** [review]
Thanks for the PR! Looking good :)

Left two small comments/questions.

File: README.md (hunk 2 above)
Comment (KnechtionsCoding) [reply README.md:177]: Yes, I can update that.

File: pkg/resources/jobs/runner.go (hunk 1 above)
Comment (KnechtionsCoding) [reply pkg/resources/jobs/runner.go:26]: Ah, good call out. I didn't mean to add that to this PR.

**Pull request merged**